src/style.css

```diff
@@ -1 +1,77 @@
+/* stylelint-disable string-quotes */
+/* stylelint-disable color-hex-case */
+/* stylelint-disable no-missing-end-of-source-newline */
 /* Styles go here */
+
+body {
+  font-family: 'Roboto', Arial, Helvetica, sans-serif;
+  font-weight: 500;
+  --accent-color: #00ACDC;
```

Comment:
More correct move variable to :root selector. 
*Note that the selector given to the ruleset defines the scope that the custom property can be used in. A common best practice is to define custom properties on the [:root](https://developer.mozilla.org/en-US/docs/Web/CSS/:root) pseudo-class, so that it can be applied globally across your HTML document.